cumulusci/cli/tests/test_service.py

```diff
@@ -103,7 +103,10 @@ def test_service_connect__attr_with_default_value():
     # but input of an empty line accepts the default.
     assert "attr (example) [PRESET]: " in result.output
     service_config = runtime.keychain.get_service("test", "test-alias")
-    assert service_config.attr == "PRESET"
+    with mock.patch("warnings.warn") as w:
+        assert service_config.lookup("attr") == "PRESET"
+        assert service_config.attr == "PRESET"
+        assert len(w.mock_calls) == 1
```

Comment:
Maybe use pytest.warns? https://docs.pytest.org/en/6.2.x/warnings.html#warns

Reply:
Done in ee235fe01